response/valid_status_code_old.go

```diff
@@ -0,0 +1,75 @@
+// +build !go1.11
+
+package response
+
+import (
+	"net/http"
+)
+
+var validStatusCode = map[int]bool{
```

Comment:
Why we have two `validStatusCode`?

Reply:
> The Status-Code element is a 3-digit integer
According to this, I think we should check it's `>= 100 && <= 999`?

Reply:
Yes, I think it will be great.